test/orange/surf/SoftSurfaceEqual.test.cc

```diff
@@ -192,6 +192,64 @@ TEST_F(SoftSurfaceEqualTest, general_quadric)
         softeq_(ref, SurfaceTranslator(Translation{{large, 0, 0}})(ref)));
 }
 
+TEST_F(SoftSurfaceEqualTest, involute)
```

Comment:
Since we're reordering the surface type enum for consistency, we might as well update the ordering here too...